install/01_clusterautoscaler.crd.yaml

```diff
@@ -2,37 +2,52 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
   name: clusterautoscalers.autoscaling.openshift.io
 spec:
   group: autoscaling.openshift.io
   names:
     kind: ClusterAutoscaler
-    listKind: ClusterAutoscalerList
     plural: clusterautoscalers
-    singular: clusterautoscaler
   scope: Cluster
   subresources:
     status: {}
   validation:
     openAPIV3Schema:
       properties:
         apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
           type: string
         kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
           type: string
         metadata:
           type: object
         spec:
+          description: Desired state of ClusterAutoscaler resource
           properties:
             maxPodGracePeriod:
+              description: Gives pods graceful termination time before scaling down
               format: int32
               type: integer
             podPriorityThreshold:
+              description: 'To allow users to schedule "best-effort" pods, which shouldn''t
+                trigger Cluster Autoscaler actions, but only run when there are spare
+                resources available, More info: https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#how-does-cluster-autoscaler-work-with-pod-priority-and-preemption'
               format: int32
               type: integer
             resourceLimits:
+              description: Constraints of autoscaling resources
               properties:
                 cores:
+                  description: Minimum and maximum number of cores in cluster, in
+                    the format :. Cluster autoscaler will not scale the cluster beyond
```

Comment:
in the format?

Reply:
https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-are-the-parameters-to-ca

Reply:
changed to `<min>:<max>`